generator.py

```diff
@@ -10,6 +12,9 @@
 NEED_UPDATE = 1
 FAILURE = 2
 
+DEFAULT_GIR_FILES = 'gir-files'
```

Comment:
What do you think about `DEFAULT_GIR_FILES_FOLDER` instead?

Reply:
This is named after "--gir-files". Would only rename it if we rename the option.

Reply:
And `--gir-files-dir` would probably be clearer

Reply:
Then please do the change here and open an issue (or even a PR) on `gir`. :)

Reply:
Wait, I'm confused. What does gir have to do with this?

Reply:
If we do it consistent with `gir`, it should be `--gir-directory`

Reply:
Arf sorry, don't know why I thought you were talking about `gir`... So yes, please feel free to update the option name here.

Reply:
Fine by me!